docs/source/installation.mdx

```diff
@@ -34,18 +29,18 @@ You can install `Apollo.framework` into your project using any of the three majo
 <a name="carthage-installation" />
 <CarthageInstallationPanel />
 
-## Adding a schema file to your target directory
+## 3. Add a schema file to your target directory
 
-You'll have to copy or [download a schema](/downloading-schema/) to your target's directory before generating code.
+For Apollo iOS to generate models for your GraphQL operations, you need a local copy of your GraphQL server's schema in JSON format. To acquire this schema, see [Downloading a schema](/downloading-schema/).
```

Comment:
This doesn't necessarily need to be in JSON, it can also be in SDL. We support a couple different extensions for this at the moment but I think the one we want to use going forward is `.graphqls` IIRC